### ISSUE TITLE
Error downloading util.sh in the latest version on MacOS

### ISSUE THREAD

**Issue by norbertschuler** [labbots/google-drive-upload#82]
I am currently using release 2.7 by using `bash <(curl --compressed -s https://raw.githubusercontent.com/labbots/google-drive-upload/master/install.sh -R v2.7`  working well. I tested downloading the latest version with `bash <(curl --compressed -s https://raw.githubusercontent.com/labbots/google-drive-upload/master/install.sh)`, but then the file `.google-drive-upload/bin/util.sh` just contained the following:

```
Bad Request
#292cd6b2064baae434cd242b3989887a4ea96cd6>[BUGFIX]</a> <span class=issue-keyword tooltipped tooltipped-se aria-label=This commit closes issue #80.><a data-pjax=true title=
```

My environment is the latest MacOS. Here is the debug log from the update install script: [debug.log](https://github.com/labbots/google-drive-upload/files/4872092/debug.log)

**Comment by Akianonymus** [comment]
yeah, waiting for labbots


https://github.com/labbots/google-drive-upload/pull/81#issuecomment-653527116

or i will eventually make a new pull request

github is acting weird these days

**Comment by Akianonymus** [comment]
https://github.com/labbots/google-drive-upload/pull/83

did the fixes

and added steps in ci to prevent this type of mistakes in future

**Comment by labbots** [comment]
@Akianonymus @norbertschuler The PR is merged and I have created a new release v2.7.3

**Comment by norbertschuler** [comment]
v2.7.3 works well for me
